### PR TITLE
Initial scores listing working state

### PR DESCRIPTION
**Pull request by tommybb**

### CODE REVIEW

File: mirage/factories/pupil.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Factory, association } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  name: 'Szymon Kolecki',
+  birthdate: new Date(),
```
Comment (cs3b) [inline mirage/factories/pupil.js:5]: it's usually better to have dynamic date

```` js
birthdate() {
  return new Date();
},
````

**Review by cs3b — CHANGES_REQUESTED** [review]
Elso have a look at ESLint errors - just by running tests

File: TIL.mb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- using resource in mirage config file let us forget about single actions (show, create etc.) definition
+
+# Use components
+
+- moving html code from templates to components can make the code more clear
+
+# Use data served by Mirage
+
+- we need to define object in routes which in we return data found in store
```
Comment (cs3b) [inline TIL.mb:17]: it's good to go through [mirage documentation](http://www.ember-cli-mirage.com/docs/v0.3.x/)

File: app/components/table-score/row/template.hbs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{{yield}}
```
Comment (cs3b) [inline app/components/table-score/row/template.hbs:1]: you don't need yield? could you explain why?

File: app/models/class-room.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import DS from 'ember-data';
```
Comment (cs3b) [inline app/models/class-room.js:1]: we need to setup eslint fully, and review the project config

File: mirage/factories/class-room.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { Factory, association } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  name: '4B',
```
Comment (cs3b) [inline mirage/factories/class-room.js:4]: prefer random data - for faker or sequence

File: mirage/factories/score.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import { Factory, association } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  date: new Date(),
+  result: 1,
```
Comment (cs3b) [inline mirage/factories/score.js:5]: prefer dynamic generated data

File: mirage/factories/year.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import { Factory } from 'ember-cli-mirage';
+
+export default Factory.extend({
+  name: '2016/2017'
```
Comment (cs3b) [inline mirage/factories/year.js:4]: similar here

File: package.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "competitor",
+  "name": "scores",
   "version": "0.0.0",
```
Comment (cs3b) [inline package.json:3]: on every deploy to production it's good to bump version here

File: tests/integration/components/table-score/header/component-test.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+import { setupComponentTest } from 'ember-mocha';
+import hbs from 'htmlbars-inline-precompile';
+
+describe('Integration | Component | table score/header', function() {
+  setupComponentTest('table-score/header', {
+    integration: true
+  });
+
+  it('renders', function() {
+    // Set any properties with this.set('myProperty', 'value');
```
Comment (cs3b) [inline tests/integration/components/table-score/header/component-test.js:12]: delete dead doce

Comment (cs3b) [reply tests/integration/components/table-score/header/component-test.js:12]: would be good to spend one session on testing

File: app/components/table-score/row/template.hbs (hunk 3 above)
Comment (tommybb) [reply app/components/table-score/row/template.hbs:1]: because I will not pass any block to row template??

**Comment by tommybb** [comment]
@cs3b,  changes done in PR #4

**Pull request merged**